generate_client.sh

```diff
@@ -1,12 +1,16 @@
 #!/bin/bash
 
-rm -rf openapi
+TMP_DIR="openapi_client_generated"
+PACKAGE_NAME="_openapi_client"
+
 openapi-generator-cli generate \
     -i https://raw.githubusercontent.com/jellyfish-dev/jellyfish/main/openapi.yaml \
     -g python \
-    -o openapi \
+    -o $TMP_DIR \
     -t templates \
-    --package-name openapi_client \
-    --global-property apis,models,modelTests=false,supportingFiles
-    
-rm -rf openapi/{docs,test,.github,.openapi-generator,.gitignore,.gitlab-ci.yml,.travis.yml,.openapi-generator-ignore,git_push.sh,README.md,setup.cfg,test-requirements.txt,tox.ini}
+    --package-name jellyfish.$PACKAGE_NAME \
+    --global-property apis,models,modelTests=false,apiTests=false,modelDocs=false,apiDocs=false,supportingFiles
+
+rm -rf jellyfish/$PACKAGE_NAME
```

Comment:
This isn't a big issue since we have version control, but you could consider exiting without deleting the old _openapi_client directory if the generate command failed